tests/setup-test-env.ts

```diff
@@ -0,0 +1,13 @@
+global.mockEnv = (env: Record<string, string>): (() => void) => {
```

Comment:
Nice refacto, this new method `mockEnv`, with `restore()` calls makes specs really easier to read.
We could have imagine here to split this PR in two commits, and keep this change in first commit, to keep changes about branch more isolated... But that's already great, no need to change, your call to decide before merging it

Reply:
Actually, I needed to make this change because of interdependent mocked process env value before this PR... I was never resetting the variables and this had wrong values passed from test to test :grimacing:.

This new method was more than necessary :)